spec/models/creator_spec.rb

```diff
@@ -2,6 +2,9 @@
 require 'rails_helper'
 
 RSpec.describe Creator, type: :model do
+  before do
```

Comment:
I'm curious why this is needed?

Reply:
Order-dependent failures on this line: `end.to raise_error(ActiveRecord::RecordInvalid, /Validation failed: Viaf id already in the system/)`, where sometimes it would give the RecordInvalid error with a "translation missing" instead of that message.  There's probably something better that could/would/should be done about this one? but that's what I got to work.

Reply:
It might be good to pull this out into it's own PR then and past the failing test.  Then if anyone asks about this in the future, we can use git blame and we'll see the PR about the flapping test.

Reply:
Broke this out into its own issue - @fnibbit if you have a seed that this happens on that would be a helpful thing to add to the ticket. https://github.com/MPLSFedResearch/cypripedium/issues/482